src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheckTest.java

```diff
@@ -97,7 +97,7 @@ public void newTest() throws Exception {
     }
 
     @Test
-    public void allowedAnnotationsTest() throws Exception {
+    public void testAllowedAnnotations() throws Exception {
```

Comment:
@romani We don't have a check that verifies tests start with the word `test`?
@ngeor You are doing a bunch of changes that are unrelated to this PR. I would consider moving them to a different PR.

Reply:
No that file has almost all the tests starting with `test` except this one and a few more. I can revert if you like, or I can fix all the tests in this file to start with `test`. Whatever you want is fine by me.

Reply:
@rnveach ,

> We don't have a check that verifies tests start with the word test?

I do not remember such Check. It is kind of naming format match based on annotation ... good idea to create such Check.

@ngeor ,

> or I can fix all the tests in this file to start with test

please send us fix in separate PR, you will be welcome. No need for issue to be created, just put prefix "minor: " in commit message and do PR.

Reply:
Good to know!